Library/Formula/ansible-cmdb.rb

```diff
@@ -16,11 +16,13 @@ class AnsibleCmdb < Formula
 
   def install
     bin.mkpath
+    man1.mkpath
     inreplace "Makefile" do |s|
       s.gsub! "/usr/local/lib/${PROG}", prefix
       s.gsub! "/usr/local/bin", bin
+      s.gsub! "/usr/local/share/man/man1/", man1 + 'ansible-cmdb.1'
+      s.gsub! "mandb", "#mandb"
```

Comment:
@alexbarton Great, thanks! It may be worth using the upstream commits as `patch do` blocks so they are easier to track/remove than `inreplace`.